### ISSUE TITLE
build issue under FreeBSD  13.2

### ISSUE THREAD

**Issue by lhmwzy** [go-gitea/gitea#24186]
### Description

Can't be built under FreeBSD 13.2,the output is:
............
webpack 5.77.0 compiled successfully in 21263 ms
Running go generate...
generating bindata for migration
generating bindata for options
generating bindata for public
generating bindata for templates
CGO_CFLAGS="-O2 -g -DSQLITE_MAX_VARIABLE_NUMBER=32766" go build -v  -tags 'bindata' -ldflags '-s -w  -X "main.MakeVersion=GNU Make 4.3" -X "main.Version=1.20.0+dev-487-g6f3c162f4" -X "main.Tags=bindata"' -o gitea
code.gitea.io/gitea/modules/migration
code.gitea.io/gitea/modules/options
code.gitea.io/gitea/modules/label
code.gitea.io/gitea/modules/translation
code.gitea.io/gitea/models/admin
code.gitea.io/gitea/modules/repository
code.gitea.io/gitea/modules/templates
code.gitea.io/gitea/services/mailer
code.gitea.io/gitea/modules/context
code.gitea.io/gitea/services/migrations
code.gitea.io/gitea/routers/web/devtest
code.gitea.io/gitea/routers/web/feed
code.gitea.io/gitea/routers/install
code.gitea.io/gitea/services/task
code.gitea.io/gitea/routers/api/v1/repo
code.gitea.io/gitea/routers/web/repo
code.gitea.io/gitea/routers/web
code.gitea.io/gitea/routers
code.gitea.io/gitea/cmd
code.gitea.io/gitea
golang.org/x/sys/cpu.runtime_getAuxv·f: relocation target runtime.getAuxv not defined
gmake: *** [Makefile:757: gitea] Error 1

### Gitea Version

commit:6f3c162

### Can you reproduce the bug on the Gitea demo site?

No

### Operating System

FreeBSD com 13.2-RELEASE FreeBSD 13.2-RELEASE releng/13.2-n254617-525ecfdad597 GENERIC amd6

### How are you running Gitea?

TAGS="bindata" gmake build

### Database

PostgreSQL

**Comment by wxiaoguang** [comment]
It looks like a Golang's problem. And maybe the latest Go 1.21 supports such `getAuvx`:

https://groups.google.com/g/golang-checkins/c/FTgTgGVcOJI

**Comment by lhmwzy** [comment]
> It looks like a Golang's problem. And maybe the latest Go 1.21 supports such `getAuvx`:
> 
> https://groups.google.com/g/golang-checkins/c/FTgTgGVcOJI

#go version
go version devel go1.21-40a0986959 Wed Feb 1 22:45:14 2023 +0000 freebsd/amd64

**Comment by wxiaoguang** [comment]
Interesting, then what about 1.20 ?

**Comment by wxiaoguang** [comment]
More information (just a guess):

```
# go mod why golang.org/x/sys/cpu
# golang.org/x/sys/cpu
code.gitea.io/gitea/models/migrations/v1_14
golang.org/x/crypto/argon2
golang.org/x/sys/cpu
```

So if the `golang.org/x/crypto/argon2` package could be removed from code (only a few), then I guess there is no need for `golang.org/x/sys/cpu` when compiling, then the bug could be by-passed.

**Comment by wxiaoguang** [comment]
Then I guess this issue could be closed? Since Golang has fixed it in 1.21.
